lib/sogou/search/api/core/base_service.rb

```diff
@@ -50,7 +51,11 @@ def client
           private
 
           def service_host
-            "api.agent.sogou.com#{ENV.fetch('ENV', 'development') == 'development' ? ':8080' : ''}"
+            "api.agent.sogou.com#{environment == 'development' ? ':8080' : ''}"
+          end
+
+          def environment
```

Comment:
this is redundant 
we have these in F3D-Job
https://github.com/forward3d/f3d-job/blob/d903eb525e08642f3446f16d5ee5b497038ee7ce/lib/f3d-job.rb#L67-L81

Reply:
and this 
https://github.com/forward3d/f3d-job/blob/d903eb525e08642f3446f16d5ee5b497038ee7ce/lib/f3d/job/framework.rb#L28

Reply:
@babakgh it's a separate public gem that doesn't have f3d-job in dependencies.